### PR TITLE
add props to etlMapping.json

### PR DESCRIPTION
**Pull request by mbkranz**
Added new properties previously added to data dictionary into the etlMapping.json

### CODE REVIEW

**Review by paulineribeyre — COMMENTED** [review]
I deployed this change to QA so you can test there before we merge this: https://github.com/uc-cdis/gitops-qa/commit/4cf39edde6d76f4fce713d109dd934244e669f73

Screen Shot 2021-08-19 at 3 03 55 PM

File: jcoin.datacommons.io/etlMapping.yaml [hunk 1]
```diff
@@ -6,8 +6,10 @@ mappings:
     props:
       - name: submitter_id
       - name: project_id
+      - name: quarter_recruited
+      - name: role_in_project
     parent_props:
-      - path: protocols[protocol]
+      - path: protocols[protocol_submitter_id:submitter_id]
```
[the remark this answers is not in the document]
Comment (mbkranz) [reply jcoin.datacommons.io/etlMapping.yaml]: Thanks for the screenshot -- I was using the protocol_submitter_id as my human readable column so this is actually good. I'll let you know after I test gitops config so we can merge

[the remark this answers is not in the document]
Comment (mbkranz) [reply jcoin.datacommons.io/etlMapping.yaml]: QQ - so for  https://qa-jcoin.planx-pla.net I need to wait for the automatic ETL process right? I I think it happens at 7pm?
image

[the remark this answers is not in the document]
Comment (paulineribeyre) [reply jcoin.datacommons.io/etlMapping.yaml]: oh, you don't have access to data in QA. woops - i can grant you access manually

[the remark this answers is not in the document]
Comment (paulineribeyre) [reply jcoin.datacommons.io/etlMapping.yaml]: or we could merge this. it's a non-breaking change so it doesn't matter if there's extra fields that we don't use yet while you're testing the portal config

[the remark this answers is not in the document]
Comment (mbkranz) [reply jcoin.datacommons.io/etlMapping.yaml]: Hmm I'm fine either way -- whatever is best -- although eventually I'll have to submit to qa-gitops first so prob will need access to data to make changes? 🤔

[the remark this answers is not in the document]
Comment (mbkranz) [reply jcoin.datacommons.io/etlMapping.yaml]: but maybe this time we can go with just merging this then in future I can get access to data -- do you need to grant me access for every deployment in QA?

[the remark this answers is not in the document]
Comment (paulineribeyre) [reply jcoin.datacommons.io/etlMapping.yaml]: if you're going to need access again, i can grant you access permanently

[the remark this answers is not in the document]
Comment (paulineribeyre) [reply jcoin.datacommons.io/etlMapping.yaml]: @mbkranz you should have access to a "TEST-JCOIN" project now

[the remark this answers is not in the document]
Comment (mbkranz) [reply jcoin.datacommons.io/etlMapping.yaml]: That would be great, thanks!

**Review by paulineribeyre — APPROVED** [review]

File: jcoin.datacommons.io/etlMapping.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (paulineribeyre) [reply jcoin.datacommons.io/etlMapping.yaml]: @mbkranz the project is empty - feel free to submit what you need. I can run the ETL once you're done

[the remark this answers is not in the document]
Comment (mbkranz) [reply jcoin.datacommons.io/etlMapping.yaml]: @paulineribeyre  Sorry about this -- I wasn't able to find the "TEST-JCOIN" project evening (or JCOIN-TEST as the project is called in PROD). I also wasn't able to submit to the dev-TEST project listed).  I could be doing something wrong though so next time I'll try again.

Just for this time -- could we just merge this PR with PROD and then I'll test config locally there (and then I'll submit the PR for gitops.json on Monday to the QA env after internal review)?

[the remark this answers is not in the document]
Comment (paulineribeyre) [reply jcoin.datacommons.io/etlMapping.yaml]: @mbkranz I think I forgot to run the sync job after granting you access yesterday, my bad. Let me know if you still don't see it

[the remark this answers is not in the document]
Comment (mbkranz) [reply jcoin.datacommons.io/etlMapping.yaml]: no worries! yup I see I have access now thanks.

**Pull request merged**